src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -92,6 +95,12 @@ public StreamingResponseBody streamEvents(
 
         return outputStream -> {
 
+            if  (floodService.isConsumptionBlocked(eventTypeName)) {
```

Comment:
Yes, I think so. There is a possibility to block by event type or owning application.